Corzbank.Helpers/Validations/ValidateUser.cs

```diff
@@ -21,22 +21,30 @@ public ValidateUser(ILogger<ValidateUser> logger, UserManager<User> userManager)
 
         public bool FirstNameValid(User user)
         {
-            const string expression = "[a-zA-Z]{0,20}$";
-
-            if (StringToCheck(user.Firstname, expression))
+            if (user.Firstname != null)
```

Comment:
In case Firstname is whitespcae (" "), validation will not pass, instead of checking on null, use string.IsNullOrWhiteSpace

Reply:
resolved